### PR TITLE
Path fixed

### PR DESCRIPTION
**Pull request by carlosreig**
Path fixed since TwigBridge v2.7 changed the folder structure

### CODE REVIEW

**Comment by xabbuh** [comment]
Good catch @carlosreig. However, I think we should do better than hardcoding paths in the `vendor` directory.

What about using reflection here to determine the path to the `TwigEngine` class which reside in the top level directory of the TwigBridge?

**Comment by carlosreig** [comment]
Cool! Good idea. I will work on it.

**Comment by carlosreig** [comment]
Changed the hardcoded path by a ReflectionClass of the AppVariable class located in the TwigBridge library.
Used AppVariable instead of TwigEngine because TwigEngine has dependencies on the Templating component that is not needed if using the Form component outside Symfony framework.

**Comment by WouterJ** [comment]
:+1:

File: components/form/introduction.rst [hunk 1]
```diff
@@ -189,10 +189,10 @@ to bootstrap or access Twig and add the :class:`Symfony\\Bridge\\Twig\\Extension
     $defaultFormTheme = 'form_div_layout.html.twig';
 
     $vendorDir = realpath(__DIR__.'/../vendor');
-    // the path to TwigBridge so Twig can locate the
+    // the path to TwigBridge library so Twig can locate the
     // form_div_layout.html.twig file
-    $vendorTwigBridgeDir =
-        $vendorDir.'/symfony/twig-bridge/Symfony/Bridge/Twig';
+    $appVariableReflection = new \ReflectionClass( '\Symfony\Bridge\Twig\AppVariable' );
```
Comment (xabbuh) [inline components/form/introduction.rst]: Can you remove the spaces around the argument?

**Comment by carlosreig** [comment]
Done! The spaces around the parameter are removed.

**Comment by WouterJ** [comment]
Thank you Carlos! This is a very flexible solution to fix the hardcoded path :)

**Pull request closed without merging**